.changeset/new-cooks-change.md

```diff
@@ -0,0 +1,5 @@
+---
+"wrangler": patch
+---
+
+add mysql configuration options to hyperdrive
```

Comment:
Could be more descriptive here. These end up in the release notes. Something like "Enable the creation of MySQL Hypedrive configs via the Wrangler CLI."